website/client/src/components/header/notificationsDropdown.vue

```diff
@@ -117,14 +117,14 @@ import * as quests from '@/../../common/script/content/quests';
 import { hasCompletedOnboarding } from '@/../../common/script/libs/onboarding';
 import notificationsIcon from '@/assets/svg/notifications.svg';
 import MenuDropdown from '../ui/customMenuDropdown';
-import MessageCount from './messageCount';
+import MessageCount from './messageCount.functional';
```

Comment:
This is interesting, what's the rationale for making this odd little change?

Reply:
Not by my choice xD

in Nov. 2022 I had to rename them https://github.com/HabitRPG/habitica/pull/14262/commits/e25922f8b33f3dad2b13110d6fdbaba2a67d87f0  otherwise the compiler not let me having them 😬  - gonna try if the current versions of vue (the ones we using now) allow it again

Reply:
Have to rollback to use `.functional` again

I couldn't find any documentation of vue of why its needed but without - the functional component seems to not get a vm object ...